### ISSUE TITLE
generateInputOnly doesn't work as expected

### ISSUE THREAD

**Issue by hliang** [rwdavies/STITCH#1]
It says: `#' @param generateInputOnly Whether to just generate input data then quit`, but when I ran stitch with `generateInputOnly=TRUE`, it continued and finished all the following imputation steps. 

I had to make it stop by inserting the these lines after `generate_or_refactor_input(...)` in file `STITCH/R/functions.R`.
```
## quit after input .RDate files are generated
if(generateInputOnly) {
  return(NULL)
}
```
BTW, I was running version 1.2.4.

**Comment by RobertDaviesGenomics** [comment]
Thanks, you're right. I must have broken that while re-factoring the input generation, and didn't have a test to check that it properly stopped at that point. I'll add a patch and test in the upcoming days

**Comment by rwdavies** [comment]
Thanks, you're right. I must have broken that while re-factoring the input generation, and didn't have a test to check that it properly stopped at that point. I'll add a patch and test in the upcoming days

**Comment by rwdavies** [comment]
This is fixed now with v1.3.1
